docs/flows.md

```diff
@@ -23,16 +28,16 @@ Input and output schemas for flows can be defined using `zod`.
 import { defineFlow } from '@genkit-ai/flow';
 import * as z from 'zod';
 
-export const myFlow = defineFlow(
+export const menuSuggestionFlow = defineFlow(
   {
-    name: 'myFlow',
-    inputSchema: z.object({ subject: z.string() }),
+    name: 'menuSuggestionFlow',
+    inputSchema: z.object({ cuisine: z.string() }),
     outputSchema: z.string(),
   },
   async (input) => {
-    const output = doSomethingCool(input.subject);
+    const suggestion = makeMenuSuggestion(input.cuisine);
 
-    return output;
+    return llmResponse.text();
```

Comment:
`suggestion.text()`?

Reply:
Actually, simply `suggestion`? I don't think this code is calling an LLM. `makeMenuSuggestion` appears to be a user defined function call?

Reply:
Yeah this was a copy-pasta error, it should just be `suggestion`. Done.